wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -903,4 +921,124 @@ public static function setIndianStateOptions(string $entity, string $field, arra
 
   }
 
+  /**
+   * This hook is called after the database write on a custom table.
+   *
+   * @param string $op
+   *   The type of operation being performed.
+   * @param string $objectName
+   *   The custom group ID.
+   * @param int $objectId
+   *   The entityID of the row in the custom table.
+   * @param object $objectRef
+   *   The parameters that were sent into the calling function.
+   */
+  public static function mailNotificationToMMT($op, $groupID, $entityID, &$params) {
+    if ($op !== 'create') {
+      return;
+    }
+
+    if (!($goonjField = self::findOfficeId($params))) {
+      return;
+    }
+
+    $goonjFieldId = $goonjField['value'];
+    $vehicleDispatcheId = $goonjField['entity_id'];
+
+    $collectionSourceVehicleDispatche = EckEntity::get('Collection_Source_Vehicle_Dispatch', FALSE)
+      ->addSelect('Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id')
+      ->addWhere('id', '=', $vehicleDispatcheId)
+      ->execute()->first();
+
+    $collectionCampId = $collectionSourceVehicleDispatche['Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id'];
```

Comment:
Please fix in all the places
```suggestion
    $collectionCampId = $collectionSourceVehicleDispatch['Camp_Vehicle_Dispatch.Collection_Camp_Intent_Id'];
```